### ISSUE TITLE
Migrate to API-Key instead User & Password authentication

### ISSUE THREAD

**Issue by Judis** [minical/channex-integration#1]
Hi!

I would like to suggest one small change at your Channex integration library. Right now you use `api/v1/sing_in` method with `user` & `password` arguments, but this method will be deprecated soon and will be excellent if you can make changes and migrate to `API-Key` based auth.

All required information you can find here - https://docs.channex.io/api-v.1-documentation/api-reference#api-key-access and https://docs.channex.io/application-documentation/api-key-access.

Basically, with API Key access, you can remove all logic associated with Channex auth and just keep API key and use it to sign requests. API Key should be provided at headers as `user-api-key`.
